scripts/README.md

```diff
@@ -0,0 +1,63 @@
+# Development scripts for Kubernetes documentation
+
+## Requirements
+
+Some of those scripts have external requirements you can install them using the following commands:
+
+python3 -m pip install -r requirements.txt
```

Comment:
Please code-fence this line.

```
    ```
    python3 -m pip install -r requirements.txt
    ```
```